frontend/server/src/Controllers/Problem.php

```diff
@@ -4529,6 +4542,11 @@ public static function getProblemDetailsForSmarty(
                 ),
             ]
         );
+
+        $response['smartyProperties']['payload']['problem'] += [
+            'nextSubmissionTimestamp' => $nextSubmissionTimestamp,
+        ];
```

Comment:
Qué te parece así:
```suggestion
        $response['smartyProperties']['payload']['problem'] = array_merge(
            $response['smartyProperties']['payload']['problem'],
            ['nextSubmissionTimestamp' => $nextSubmissionTimestamp]
        );
```
?
Por consistencia con L4527